src/pages/ProductsList/components/LeftSide/MapBox.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flexSort } from '../../../../styles/mixin';
+import theme from '../../../../styles/theme';
+
+const MapBox = () => {
+  return (
+    <MapBoxImage>
+      <MapImage>
+        <MapButton>지도보기</MapButton>
+      </MapImage>
+    </MapBoxImage>
+  );
+};
+
+const MapBoxImage = styled.div`
+  padding: 60px 0 0 0;
```

Comment:
```suggestion
  padding-top: 60px;
```
특정 부분만 적용되야 한다면 위와 같이 작성해주세요